### PR TITLE
handle errors

### PR DESCRIPTION
**Pull request by cubixle**
removes unused code and handles errors that weren't handled

### CODE REVIEW

File: git.go [hunk 1]
```diff
@@ -106,7 +109,9 @@ func (g *gitdb) gitPush() error {
 func (g *gitdb) gitCommit(filePath string, msg string, user *DbUser) {
 	mu.Lock()
 	defer mu.Unlock()
-	g.gitDriver.commit(filePath, msg, user)
+	err := g.gitDriver.commit(filePath, msg, user)
```
Comment (fobilow) [inline git.go:112]: here we should check if err is not nil before logging

File: db.go [hunk 2]
```diff
@@ -31,10 +30,6 @@ type SearchParam struct {
 }
 
 type gitdb struct {
-	mu sync.Mutex
```
Comment (fobilow) [inline db.go:34]: `mu` will be used by a commit I am working on so best to keep

**Review by fobilow — CHANGES_REQUESTED** [review]
See comments in review

**Comment by cubixle** [comment]
> See comments in review

they have been addressed in follow up commits and marked as resolved.

**Review by fobilow — APPROVED** [review]
Looks good. thanks

**Pull request merged**